ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -26,6 +27,12 @@ protected void configure() {
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
 		boolean enabled = true;
+		Optional<Long> lingersMs;
+		Optional<Integer> batchSize;
+		Optional<Integer> sendBufferBytes;
+		Optional<Integer> maxInFlightRequestsPerConnection;
+		Optional<Long> bufferMemory;
```

Comment:
From @charliek comment: This is due to my comments here [#15](https://github.com/SmartThingsOSS/ratpack-kafka/pull/15) . Mainly I want to only set the properties to kafka if values are set, and not maintain all of our own defaults that mirror the driver. Having the set be optional seems a little odd to me as well